### PR TITLE
MAGECLOUD-4887: Implement wizard for split Db  'wizard:split-db-state'

### PR DESCRIPTION
**Pull request by NadiyaS**
https://magento2.atlassian.net/browse/MAGECLOUD-4887

### Description
Implement CLI command which provides info about ability to split DB and whether DB was already split or not.
### Manual testing scenarios
Run command `wizard:split-db-state` on next cloud environments:
**1.** On any env, which does not support split DB. 
  Result: 
```
DB is not split:
- DB cannot be split on this environment
```
**2.** On env which support split DB.
  Result: 
```
DB is not split:
- You may split DB using SPLIT_DB variable in .magento.env.yaml file
```
**3.** On env which support split DB. And quote type was already split
  Result: 
```
DB is already split with type(s): quote
```
**4.** On env which support split DB. And quote and sales types were already split
  Result: 
```
DB is already split with type(s): quote, sales
```

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Review by billygilbert — DISMISSED** [review]

File: src/Command/Wizard/SplitDbState.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Verifies whether DB was split or not
+ *
+ * @api
+ */
+class SplitDbState extends Command
+{
+    const NAME = 'wizard:split-db-state';
```
Comment (arhiopterecs) [inline src/Command/Wizard/SplitDbState.php:30]: Add a description

File: src/Command/Wizard/SplitDbState.php [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+    /**
+     * @var OutputFormatter
+     */
+    private $outputFormatter;
```
Comment (arhiopterecs) [inline src/Command/Wizard/SplitDbState.php:37]: Add a description

File: src/Command/Wizard/SplitDbState.php [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+    /**
+     * @var ConfigReader
+     */
+    private $configReader;
```
Comment (arhiopterecs) [inline src/Command/Wizard/SplitDbState.php:44]: Add a description

File: src/Command/Wizard/SplitDbState.php [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+    /**
+     * Factory for creation database configurations
+     *
+     * @var RelationshipConnectionFactory
+     */
+    private $connectionDataFactory;
+
+    /**
+     * SplitDb constructor.
```
Comment (arhiopterecs) [inline src/Command/Wizard/SplitDbState.php]: Please, delete this line

**Review by arhiopterecs — CHANGES_REQUESTED** [review]

File: src/Command/Wizard/SplitDbState.php (hunk 1 above)
Comment (NadiyaS) [reply src/Command/Wizard/SplitDbState.php:30]: done

File: src/Command/Wizard/SplitDbState.php (hunk 2 above)
Comment (NadiyaS) [reply src/Command/Wizard/SplitDbState.php:37]: done

File: src/Command/Wizard/SplitDbState.php (hunk 3 above)
Comment (NadiyaS) [reply src/Command/Wizard/SplitDbState.php:44]: done

File: src/Command/Wizard/SplitDbState.php (hunk 4 above)
Comment (NadiyaS) [reply src/Command/Wizard/SplitDbState.php]: done

**Review by arhiopterecs — APPROVED** [review]

**Review by arhiopterecs — APPROVED** [review]

**Review by billygilbert — APPROVED** [review]

**Comment by andriyShevtsov** [comment]
QA approved

**Pull request merged**